File: FinanceService/Models/Expense.cs
namespace FinanceService.Models
{
    public class Expense
    {
        public int Id { get; set; }
        public int TravelPlanId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Category { get; set; } = string.Empty;
        public decimal Amount { get; set; }
        public DateTime ExpenseDate { get; set; }
        public string? Description { get; set; }
    }
}

File: travel-planner-frontend/src/models/Expense.ts
export interface Expense{
    id: number;
    travelPlanId: number;
    name: string;
    category: string;
    amount: number;
    expenseDate: string;
    description?: string;
}

export interface CreateExpenseDto{
    name: string;
    category: string;
    amount: number;
    expenseDate: string;
    description?: string;
}

export interface BudgetSummary{
    travelPlanId: number;
    totalBudget: number;
    totalExpenses: number;
    remainingBudget: number;
}